peers.list

```diff
@@ -0,0 +1,4 @@
+did:key:z6MkmesFj9djBn5dyH4vMEAjZeBjCb1BYhKgBMtC2EeeknHn
```

Comment:
Should we add comments to know whose keys were in the list?

Reply:
sure, do add what they are when you change the name.
the last entry is for the user `meed`

Reply:
added comment